README.md

```diff
@@ -41,6 +41,12 @@ Unlike traditional speech recognition systems that rely on continuous audio stre
     ```
     This command captures audio from the microphone and sends it to the server for transcription. It uses the same options as the previous command, enabling the multilingual feature and specifying the target language and task.
 
+    - To trasncribe from a HLS stream:
+    ```python
+      client = TranscriptionClient(host, port, is_multilingual=True, lang="en", translate=False) 
+      client(hls_url="http://domain.url/playlist.m3u8") 
```

Comment:
Is this a test url? because when I test this out; this has no transcript at all.

Reply:
@makaveli10 the test link is in the first comment [here](https://github.com/collabora/WhisperLive/pull/73#issue-2034040337).

What you referenced is a non FQDN sample url for the README.md purposes.

Reply:
Given that you're not providing a file in this example on the README.md

  ```
from whisper_live.client import TranscriptionClient
  client = TranscriptionClient("localhost", 9090, is_multilingual=True, lang="hi", translate=True)
  client(audio_file_path)
```

We should keep consistency for users on the same format.

Reply:
@zoq Are we adding then some samples into the README.md or leaving it as it is? Let me know.

Reply:
@jhormigo thanks for pointing that out. I'll add a sample to the repo and mention to the readme. Meanwhile, can you please do the same for HLS stream as well?

Reply:
@makaveli10 I've added it now. You can probably merge this now.